samples/subsys/ipc/openamp/README.md

```diff
@@ -0,0 +1,72 @@
+OpenAMP Example Application
+===========================
+
+Overview
+--------
+
+This application demostrates how to use OpenAMP with Zephyr. It is designed to
+be an OpenAMP counterpart to the RPMsg-Lite example for the LPCXpresso5411x
+chip, to be used for comparison and benchmarking of the two RPC libraries.
+
+Building this application
+-------------------------
+
+Building and running this application requires that libmetal and OpenAMP is
+compiled first. Clone the libmetal and OpenAMP repositories in an appropriate
+location:
+
```

Comment:
Use the code-block directive to get this formatted correctly:

    .. code-block:: bash

       git clone https://github.com/OpenAMP/libmetal.git
       git clone https://github.com/OpenAMP/open-amp.git